modules/Oculus_module/src/OculusModule.cpp

```diff
@@ -416,6 +418,73 @@ bool OculusModule::configure(yarp::os::ResourceFinder& rf)
     }
     m_oculusHeadsetPoseInertial.resize(6, 0.0);
 
+    //Reset the cameras if necessary
+    bool resetCameras = generalOptions.check("resetCameras", yarp::os::Value(false)).asBool();
+    yInfo() << "[OculusModule::configure] Reset camera: " << resetCameras;
+    if (resetCameras)
+    {
+        auto cameraReset = [](const std::string& cameraPort, const std::string& localPort) -> bool
```

Comment:
Is there any reason you decided to use lambda function instead of a private member function of the class? I a saying that, since the code of this function is inside `configure` member function, and the code here is long in this function and adding this lambda function make it longer even. Also, it reduces the readability.
In any case, in line with other private functions we have such as `configureJoypad` and `configureOculus` please use `configureCamera` private member function.

Reply:
Moved to a method ``resetCamera`` (not ``configure`` since I am not really configuring anything) in https://github.com/robotology/walking-teleoperation/pull/45/commits/16413ff242361bb6c487e4d941bcee5a7bfcbb2e